.github/workflows/ploomber-cloud.yaml

```diff
@@ -26,4 +26,4 @@ jobs:
         env:
           PLOOMBER_CLOUD_KEY: ${{ secrets.PLOOMBER_CLOUD_KEY }}
         run: |
-          ploomber-cloud deploy
+          ploomber-cloud deploy --watch
```

Comment:
add a little comment explaining what the watch flag does and a link to the docs so they can learn more

Reply:
added [comment](https://github.com/edublancas/cloud-template/pull/1/files#diff-cfd75666a1a6780b4eb2d6665de3e934c8fffcdb680c1b8e604656f619a850cbR30-R31)